WeakAurasOptions/OptionsFrames/TextEditor.lua

```diff
@@ -108,57 +207,69 @@ local function settings_dropdown_initialize(frame, level, menu)
     }
     UIDropDownMenu_AddButton(item)
   end
-  UIDropDownMenu_AddButton({
-    text = L["Bracket Matching"],
-    isNotRadio = true,
-    checked = function()
-      return WeakAurasSaved.editor_bracket_matching
-    end,
-    func = function()
-      WeakAurasSaved.editor_bracket_matching = not WeakAurasSaved.editor_bracket_matching
-    end
-  })
+  UIDropDownMenu_AddButton(
+    {
+      text = L["Bracket Matching"],
+      isNotRadio = true,
+      checked = function()
+        return WeakAurasSaved.editor_bracket_matching
+      end,
+      func = function()
+        WeakAurasSaved.editor_bracket_matching = not WeakAurasSaved.editor_bracket_matching
+      end
+    }
+  )
 end
 
 local function ConstructTextEditor(frame)
-  local group = AceGUI:Create("InlineGroup");
-  group.frame:SetParent(frame);
-  group.frame:SetPoint("BOTTOMRIGHT", frame, "BOTTOMRIGHT", -17, 12);
-  group.frame:SetPoint("TOPLEFT", frame, "TOPLEFT", 17, -10);
-  group.frame:Hide();
-  group:SetLayout("fill");
-
-  local editor = AceGUI:Create("MultiLineEditBox");
-  editor:SetWidth(400);
-  editor.button:Hide();
-  local fontPath = SharedMedia:Fetch("font", "Fira Mono Medium");
-  if(fontPath) then
-    editor.editBox:SetFont(fontPath, 12);
+  local group = AceGUI:Create("InlineGroup")
+  group.frame:SetParent(frame)
+  group.frame:SetPoint("BOTTOMRIGHT", frame, "BOTTOMRIGHT", -17, 12)
+  group.frame:SetPoint("TOPLEFT", frame, "TOPLEFT", 17, -10)
+  group.frame:Hide()
+  group:SetLayout("fill")
+
+  local editor = AceGUI:Create("MultiLineEditBox")
+  editor:SetWidth(400)
+  editor.button:Hide()
+  local fontPath = SharedMedia:Fetch("font", "Fira Mono Medium")
+  if (fontPath) then
+    editor.editBox:SetFont(fontPath, 12)
   end
-  group:AddChild(editor);
-  editor.frame:SetClipsChildren(true);
+  group:AddChild(editor)
+  editor.frame:SetClipsChildren(true)
 
   -- The indention lib overrides GetText, but for the line number
   -- display we ned the original, so save it here.
-  local originalGetText = editor.editBox.GetText;
+  local originalGetText = editor.editBox.GetText
   set_scheme()
   IndentationLib.enable(editor.editBox, color_scheme, 4)
 
-  local cancel = CreateFrame("Button", nil, group.frame, "UIPanelButtonTemplate");
-  cancel:SetScript("OnClick", function() group:CancelClose() end);
-  cancel:SetPoint("BOTTOMRIGHT", -27, 13);
+  local cancel = CreateFrame("Button", nil, group.frame, "UIPanelButtonTemplate")
```

Comment:
Something in your IDE must be wrongly configured, it shouldn't reformat existing code. Though don't worry about it for this PR. Removing those unncessary changes is not necessary imho.

Reply:
Apologies. My initial PR had a lot of weird indenting that looked fine in my IDE but not in the commits so I did a rightclick+reformat on the file. I'll avoid this in future.